### PR TITLE
DEV: update module API for 7.4 GA

### PR DESCRIPTION
**Pull request by dwdougherty**

### CODE REVIEW

File: content/develop/reference/modules/modules-api-ref.md [hunk 1]
```diff
@@ -3840,7 +3840,7 @@ AOF rewrite, and so forth). In this section we define this API.
 
 Register a new data type exported by the module. The parameters are the
 following. Please for in depth documentation check the modules API
-documentation, especially [https://redis.io/docs/latest/develop/reference/modules/modules-native-types/](https://redis.io/docs/latest/develop/reference/modules/modules-native-types/).
+documentation, especially [https://redis.io/topics/modules-native-types](https://redis.io/topics/modules-native-types).
```
Comment (dwdougherty) [inline content/develop/reference/modules/modules-api-ref.md:3843]:
```suggestion
documentation, especially [https://redis.io/docs/latest/develop/reference/modules/modules-native-types/](https://redis.io/docs/latest/develop/reference/modules/modules-native-types/).
```

File: content/develop/reference/modules/modules-api-ref.md [hunk 2]
```diff
@@ -4489,7 +4489,7 @@ latency-monitor-threshold.
 ## Blocking clients from modules
 
 For a guide about blocking commands in modules, see
-[https://redis.io/docs/latest/develop/reference/modules/modules-blocking-ops/](https://redis.io/docs/latest/develop/reference/modules/modules-blocking-ops/).
+[https://redis.io/topics/modules-blocking-ops](https://redis.io/topics/modules-blocking-ops).
```
Comment (dwdougherty) [inline content/develop/reference/modules/modules-api-ref.md:4492]:
```suggestion
[https://redis.io/docs/latest/develop/reference/modules/modules-blocking-ops/](https://redis.io/docs/latest/develop/reference/modules/modules-blocking-ops/).
```

**Review by dwdougherty — COMMENTED** [review]
Coupla link changes

**Review by kaitlynmichael — APPROVED** [review]

**Review by rrelledge — APPROVED** [review]

**Comment by dwdougherty** [comment]
Thank you @kaitlynmichael and @rrelledge !

**Pull request merged**